src/Calculator/Views/GraphingCalculator/KeyGraphFeaturesPanel.xaml

```diff
@@ -152,7 +152,7 @@
             <RowDefinition Height="Auto"/>
             <RowDefinition Height="*"/>
         </Grid.RowDefinitions>
-        <Grid>
+        <Grid Margin="0,0,1,0" Background="{ThemeResource TextControlBackground}">
```

Comment:
the background looks great! But I would prefer without the margin so the "header bar" goes up to the side of the window.

Reply:
The margin is there to match the margin of the original EquationTextBox, without it looks like the textbox gets slightly bigger for no reason when transitioning between the two screens.

Reply:
Let's use `1,0` in this case to match the textboxes